.changeset/forty-steaks-scream.md

```diff
@@ -0,0 +1,57 @@
+---
+'@shopify/shopify-app-remix': patch
+---
+
+Adding `AppProvider` component to abstract Shopify-specific app setup on the frontend side.
+This makes it easier for apps to set up the components it needs to work with Shopify.
+
+To make use of this in the Remix app template, you can update your `app/routes/app.jsx` file's `App` component from
+
+```ts
+export default function App() {
+  const {apiKey, polarisTranslations} = useLoaderData();
+
+  return (
+    <>
+      <script
+        src="https://cdn.shopify.com/shopifycloud/app-bridge.js"
+        data-api-key={apiKey}
+      />
+      <ui-nav-menu>
+        <Link to="/app" rel="home">
+          Home
+        </Link>
+        <Link to="/app/additional">Additional page</Link>
+      </ui-nav-menu>
+      <PolarisAppProvider
+        i18n={polarisTranslations}
+        linkComponent={RemixPolarisLink}
```

Comment:
To make it clear you no longer need it, do we also want to show the link Component in this example?